CSS/auth.css

```diff
@@ -0,0 +1,23 @@
+.signup-card {
```

Comment:
cart written in text as well as icon side by side. any particular reason for it?

Reply:
Thanks for pointing it out removing it